pom.xml

```diff
@@ -43,6 +46,17 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
I am not 100% sure that we need to ad the jacoco plugin as a dependency. Usually the `dependencies` tag isused to list dependencies that are needed by the application or the tests. I suspect that removing this would work, after all the plugins are part of the build process and the dependencies are part of the application itself.